### PR TITLE
Add staging and release repo definitions

### PR DESCRIPTION
**Pull request by JPWKU**
Signed-off-by: Jeremy Phelps <jphelps@linuxfoundation.org>

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -38,7 +38,7 @@
 		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 		<java.version>1.8</java.version>
 		<nexusproxy>https://nexus.edgexfoundry.org</nexusproxy>
-		<repobasepath>content/repositories</repobasepath>
+		<repobasepath>/content/repositories</repobasepath>
```
Comment (tykeal) [inline pom.xml]: Given that you're using ${repobasepath} prefixed with a / in all later calls, I would suggest not having a leading '/' in there.

**Review by tykeal — CHANGES_REQUESTED** [review]
Repobase path should be cleaned up a little

File: pom.xml (hunk 1 above)
Comment (JPWKU) [reply pom.xml]: ah good catch thanks

**Review by jpwhitemn — APPROVED** [review]

**Review by trcox — APPROVED** [review]

**Comment by JPWKU** [comment]
@tykeal your comment re: " 

Repobase path should be cleaned up a little
"
Is this ok now?

**Review by tykeal — APPROVED** [review]

**Comment by tykeal** [comment]
@JPWKU yes, sorry about that. I forgot that since I'm an owner of the GitHub org my negative reviews block, along with GitHub not clearing reviews on updates to patches like Gerrit :-/

@trcox or @jpwhitemn whenever you want to give it a merge. I could, but I'm not officially a committer on the project ;)

**Pull request merged**